File: Demo-Wpf/Pos.Wpf/DAL/Product.cs
using System;
using System.Collections.Generic;
using System.Data.Entity;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Pos.Wpf.DAL
{
    public class Product
    {
        public string Barcode { get; set; }
        public string CatalogCode { get; set; }
        public string CatalogName { get; set; }
        public decimal Price { get; set; }
        public bool HasVat { get; set; }
    }
}

File: Demo-Java/pos/src/main/java/pos/dal/Product.java
package pos.dal;

public class Product {
    private String barcode;
    private String catalogCode;
    private String catalogName;
    private Double price;

    public Product() {
    }

    public String getBarcode() { return barcode; }
    public void setBarcode(String barcode) { this.barcode = barcode; }

    public String getCatalogCode() { return catalogCode; };
    public void setCatalogCode(String catalogCode) { this.catalogCode = catalogCode; }

    public String getCatalogName() { return catalogName; }
    public void setCatalogName(String catalogName) { this.catalogName = catalogName; }

    public Double getPrice() { return price; }
    public void setPrice(Double price) {
        this.price = price;
    }
}
